hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/record/AbstractNearCacheRecord.java

```diff
@@ -42,8 +43,9 @@
             AtomicLongFieldUpdater.newUpdater(AbstractNearCacheRecord.class, "recordState");
 
     protected long creationTime = TIME_NOT_SET;
-    protected long sequence;
+    protected MetaDataContainer metaDataContainer;
```

Comment:
This is the price for the speedup: one reference to the `MetaDataContainer` per `NearCacheRecord`